packages/playwright-core/src/server/firefox/ffBrowser.ts

```diff
@@ -236,7 +237,7 @@ export class FFBrowserContext extends BrowserContext {
           // validateBrowserContextOptions ensures correct video size.
           options: {
             ...this._options.recordVideo!.size!,
-            dir: this._options.recordVideo!.dir,
+            dir: path.resolve(process.cwd(), this._options.recordVideo!.dir),
```

Comment:
Should we fix it on the client instead? Sending a relative path over protocol sounds strange.